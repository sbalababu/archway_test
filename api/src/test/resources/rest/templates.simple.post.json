{
  "name" : "Sesame",
  "summary" : "A brief summary",
  "description" : "A longer description",
  "requester" : "john.doe",
  "compliance" : {
    "phi_data" : false,
    "pci_data" : false,
    "pii_data" : false
  }
}