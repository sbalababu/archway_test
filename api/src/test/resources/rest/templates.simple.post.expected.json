{
  "name": "Sesame",
  "summary": "A brief summary",
  "description": "A longer description",
  "behavior": "simple",
  "status": "Pending",
  "compliance": {
    "phi_data": false,
    "pci_data": false,
    "pii_data": false
  },
  "data": [
    {
      "name": "sw_sesame",
      "location": "/data/shared_workspaces/sesame",
      "size_in_gb": 250,
      "managing_group": {
        "group": {
          "distinguished_name": "cn=edh_sw_sesame,ou=heimdali,dc=jotunn,dc=io",
          "common_name": "edh_sw_sesame",
          "sentry_role": "role_sw_sesame",
          "attributes" : [
            [
              "dn",
              "cn=edh_sw_sesame,ou=heimdali,dc=jotunn,dc=io"
            ],
            [
              "objectClass",
              "group"
            ],
            [
              "objectClass",
              "top"
            ],
            [
              "sAMAccountName",
              "edh_sw_sesame"
            ],
            [
              "cn",
              "edh_sw_sesame"
            ]
          ]
        }
      },
      "readwrite_group": {
        "group": {
          "distinguished_name": "cn=edh_sw_sesame_rw,ou=heimdali,dc=jotunn,dc=io",
          "common_name": "edh_sw_sesame_rw",
          "sentry_role": "role_sw_sesame_rw",
          "attributes" : [
            [
              "dn",
              "cn=edh_sw_sesame_rw,ou=heimdali,dc=jotunn,dc=io"
            ],
            [
              "objectClass",
              "group"
            ],
            [
              "objectClass",
              "top"
            ],
            [
              "sAMAccountName",
              "edh_sw_sesame_rw"
            ],
            [
              "cn",
              "edh_sw_sesame_rw"
            ]
          ]
        }
      },
      "readonly_group" : {
        "group" : {
          "common_name" : "edh_sw_sesame_ro",
          "distinguished_name" : "cn=edh_sw_sesame_ro,ou=heimdali,dc=jotunn,dc=io",
          "sentry_role" : "role_sw_sesame_ro",
          "attributes" : [
            [
              "dn",
              "cn=edh_sw_sesame_ro,ou=heimdali,dc=jotunn,dc=io"
            ],
            [
              "objectClass",
              "group"
            ],
            [
              "objectClass",
              "top"
            ],
            [
              "sAMAccountName",
              "edh_sw_sesame_ro"
            ],
            [
              "cn",
              "edh_sw_sesame_ro"
            ]
          ]
        }
      }
    }
  ],
  "processing": [
    {
      "pool_name": "root.sw_sesame",
      "max_cores": 4,
      "max_memory_in_gb": 16
    }
  ],
  "applications": [
    {
      "name": "default",
      "consumer_group": "sesame_default_cg",
      "group": {
        "distinguished_name": "cn=sesame_default_cg,ou=heimdali,dc=jotunn,dc=io",
        "common_name": "sesame_default_cg",
        "sentry_role": "role_sesame_default_cg",
        "attributes" : [
          [
            "dn",
            "cn=sesame_default_cg,ou=heimdali,dc=jotunn,dc=io"
          ],
          [
            "objectClass",
            "group"
          ],
          [
            "objectClass",
            "top"
          ],
          [
            "sAMAccountName",
            "sesame_default_cg"
          ],
          [
            "cn",
            "sesame_default_cg"
          ]
        ]
      }
    }
  ],
  "topics": [],
  "single_user": false,
  "requester" : "john.doe"
}